common/changes/leddie24-remove-button-hero_2017-01-17-20-59.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Button: Remove Hero variant",
+      "type": "minor"
```

Comment:
This would be a major change I think. Let's keep this open for a little bit; I'll be refactoring button into its own package, and I will likely major bump then. Also @micahgodbolt  has a changelist coming regarding a button refactor, ideally we'd merge these two things and update at once.